kafka-connect-s3/src/test/java/io/confluent/connect/s3/TestWithMockedS3.java

```diff
@@ -107,18 +108,20 @@ public void tearDown() throws Exception {
   }
 
   public static Collection<Object> readRecords(String topicsDir, String directory, TopicPartition tp, long startOffset,
-                                               String extension, String zeroPadFormat, String bucketName,
-                                               AmazonS3 s3) throws IOException {
-      String fileKey = FileUtils.fileKeyToCommit(topicsDir, directory, tp, startOffset, extension, zeroPadFormat);
+                                               String extension, String zeroPadFormat, CompressionType compressionType,
```

Comment:
Let's create a method `readRecords` with same signature as the old one, that calls the new method with `CompressionType.NONE`. This will leave any avro related files unchanged here.